### PR TITLE
packaging fixes

### PR DESCRIPTION
**Pull request by QAston**

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
+let
```
Comment (jenpaff) [inline default.nix]: @QAston do we need the `default.nix` file in here?

**Comment by jenpaff** [comment]
oh btw: it seems like the previous changes broke the flow of using the CLI interactively. I did not spend any more time to fix the issues, but have put a note in the related task (https://app.clickup.com/t/258truu) since I think we have other higher priority tasks on our plate atm , do you think thats ok? @QAston

File: default.nix (hunk 1 above)
Comment (QAston) [reply default.nix]: no, we can delete it

**Comment by QAston** [comment]
Yeah, we can fix that later, if we ever get to upstreaming this

**Pull request merged**